pkg/relayer/session/session_test.go

```diff
@@ -73,9 +73,9 @@ func requireProofCountEqualsExpectedValueFromProofParams(t *testing.T, proofPara
 	}
 	supplierOperatorAddress := sample.AccAddress()
 	// Set the supplier operator balance to be able to submit the expected number of proofs.
-	feePerProof := prooftypes.DefaultParams().ProofSubmissionFee.Amount.Int64()
+	claimAndFeeGasCost := session.ClamAndProofGasCost.Amount.Int64()
```

Comment:
Got a hint on this here https://github.com/pokt-network/poktroll/blob/46a0fca66aa64c6df55016bf4bca501bebf8baed/pkg/relayer/session/claim.go#L275